README.md

```diff
@@ -48,3 +48,63 @@ Requests to cross-scheme subresources will no longer send SameSite=Lax or SameSi
 
 For example, if http://site.example embeds an image from https://site.example, SameSite=Lax and SameSite=Strict cookies will no longer be sent on the request to https://site.example, and any such cookies in the response will be ignored. 
 
+## Questions
+### How do Schemeful Same-Site and Scheme-Bound Cookies differ?
+
+"Schemeful Same-Site" and "[Scheme-Bound Cookies](https://github.com/mikewest/scheming-cookies)" are both trying to move cookies closer to an origin based security model; these two proposals complement one another and one is not a subset of the other.
+
+#### Schemeful Same-Site
+"Schemeful Same-Site", and the current version of same-site, is concerned with the browsing context in which the cookie will be sent or recieved. The browsing context can be thought of as the answer to the question "Is the request/response url the same site as the one I'm on?" More specifically "Is the request/response url 'same-site' or 'cross-site' with the current site I'm on?" 
+
+In the current world the browser context is determined by checking the registrable domain: http://site1.example is *not* same-site with http://site2.example, but http://site1.example and https://site1.example would be same-site.
+With "Schemeful Same-Site" we now consider the scheme along with the registrable domain: http://site1.example is cross-site, not same-site, with http://site2.example, http://site1.example is also cross-site with https://site1.example, but https://site1.example is same-site with https://site1.example. 
+
+#### Scheme-Bound Cookies
+"Scheme-Bound Cookies" is concerend with the connection overwhich a cookie is being sent or set.
```

Comment:
Connections is an odd way to phrase this because we may have multiple connections for an origin and multiple origins to a connection. There's even a spec (which we don't implement) for serving http origins over TLS connections ("opportunistic encryption").

I would suggest we avoid the word at all. Instead, we should talk about whether cookies are sent to particular origins or to particular URLs.

Reply:
TIL about opportunistic encryption, neat.